packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -45,6 +45,25 @@ class EditorPage {
 		return await this.driver.hasElementByAccessibilityId( 'block-list' );
 	}
 
+	// For text blocks, e.g Paragraph, Heading
+	async getTextBlockLocatorAtPosition( blockName, position = 1 ) {
+		// iOS needs a click before
+		if ( ! isAndroid() ) {
+			const textBlockLocator = `(//XCUIElementTypeButton[contains(@name, "${ blockName } Block. Row ${ position }")])`;
+			const textBlock = await waitForVisible(
+				this.driver,
+				textBlockLocator
+			);
+			await textBlock.click();
+		}
+
+		const blockLocator = isAndroid()
+			? `//android.view.ViewGroup[contains(@content-desc, "${ blockName } Block. Row ${ position }.")]/android.widget.EditText`
+			: `//XCUIElementTypeButton[contains(@name, "${ blockName } Block. Row ${ position }.")]//XCUIElementTypeTextView`;
+
+		return await waitForVisible( this.driver, blockLocator );
+	}
```

Comment:
This new function combines `getBlockAtPosition()` and `getTextViewForParagraphBlock()` that is used together into 1, which simplifies tests using paragraph block.